go.mod

```diff
@@ -1,26 +1,13 @@
 module github.com/edgexfoundry/device-sdk-go
 
 require (
-	github.com/BurntSushi/toml v0.3.1
```

Comment:
@cloudxxx8 @weichou1229 @SteveOss What's the deal with all of these go.mod changes?

Reply:
These packages are out of use. When  I resync the package, they are removed by GoLand IDE. 
I also search in the project and didn's see any dependency.